src/ctia/http/exceptions.clj

```diff
@@ -94,6 +94,14 @@
       :entity entity
       :class (.getName (class e))})))
 
+(defn realize-entity-error-handler
+  "Handle error at the realize step"
+  [^Exception e data request]
+  (logging/log! :info e (ex-message e))
```

Comment:
`ex-message` was apparently added in Clojure version 1.10: https://clojure.github.io/clojure/clojure.core-api.html but we are using Clojure 1.9: https://github.com/threatgrid/ctia/blob/master/project.clj#L31

I suggest to use the old `(.getMessage e)`.

Reply:
`ex-message` is already used in other error handlers. We will probably have to rename it when we move to Clojure 1.10